src/main/java/com/example/b2b_opportunities/Mapper/PositionMapper.java

```diff
@@ -45,6 +43,7 @@ public static PositionResponseDto toResponseDto(Position position) {
                 .responsibilities(position.getResponsibilities())
                 .hiringProcess(position.getHiringProcess())
                 .description(position.getDescription())
+                .status(position.getStatus().getName())
```

Comment:
Just like in the other fields, I think that the FO team will expect an Id instead of status name here.